packages/StatusTracker/src/components/Point/Point.styles.js

```diff
@@ -0,0 +1,108 @@
+import styled, { css } from "styled-components";
+import Button from "@paprika/button";
+import PaprikaPopover from "@paprika/popover";
+import tokens from "@paprika/tokens";
+import { fontSize, spacer } from "@paprika/stylers/lib/helpers";
+import * as types from "../../types";
+
+const passedPath = css`
+  border-bottom: 2px solid ${tokens.color.greenLighten10};
+`;
+
+const futurePath = css`
+  border-bottom: 2px dashed ${tokens.color.blackLighten20};
+`;
+
+export const PointWrapper = styled.li`
+  align-items: center;
+  display: inline-flex;
+  flex-grow: 1;
+  list-style: none;
+
+  &::after {
+    content: "";
+    display: inline-block;
+    flex-grow: 1;
+    margin: 0;
+    vertical-align: middle;
+
+    ${({ kind }) => (kind === types.kinds.PAST ? passedPath : futurePath)}
+  }
+
+  &:last-child {
+    flex-grow: 0;
+
+    ::after {
+      content: none;
+    }
+  }
+`;
+
+const basicPoint = css`
+  border-radius: 50%;
+  height: ${spacer(2)};
+  width: ${spacer(2)};
+`;
+
+const pastPoint = css`
+  background-color: ${tokens.color.greenLighten10};
+`;
+
+const futurePoint = css`
+  background-color: ${tokens.color.blackLighten70};
+  border: 2px solid ${tokens.color.blackLighten20};
+`;
+
+export const Point = styled.div`
```

Comment:
as standard this should be:

styled.div(({ kind }) => {
    switch (kind) {
      case types.kinds.PAST:
        return css`
          ${basicPoint}
          ${pastPoint}
        `;
      case types.kinds.CURRENT:
        return css`
          > [data-pka-anchor="popover"] > [data-pka-anchor="button"] {
            margin-top: -${tokens.spaceSm};
          }
        `;
      case types.kinds.FUTURE:
        return css`
          ${basicPoint}
          ${futurePoint}
        `;
      default:
        return null;
    }
  }} 
})